### ISSUE TITLE
The game won't use dedicated GPU

### ISSUE THREAD

**Issue by emotionalrazor** [shinyquagsire23/OpenJKDF2#130]
I am using a laptop to play the game and even after switching everything to use the dedicated gpu in my nvidia control panel, the game stll won't use it. Any help with this?

**Comment by katastic** [comment]
https://github.com/shinyquagsire23/OpenJKDF2/blob/9f7dc1e5b0cf2152004a13e8c6c3da10f0be333f/src/Win95/stdDisplay.c#L60

looks like it sets to first numerated display regardless of function argument or any configurable setting.

The user might be able to go into their laptop BIOS and configure which "videocard" the BIOS sets as the primary / first display, or even disabling the onboard video (I almost always do that anyway on my laptops as it causes nothing but problems).

**Comment by shinyquagsire23** [comment]
@katastic  stdDisplay is mostly stubbed/dead code with the SDL2 renderer, since everything scales to the window size and palettes are all in shaders.

OpenGL 3.3 doesn't really have any conception of picking video cards either. When I implement a Vulkan renderer I have an Optimus laptop and an eGPU I can test it with though, but I seem to recall that getting apps to launch with the GPU on Linux required setting some environment vars (not sure on Windows).

**Comment by katastic** [comment]
Oh, it's OpenGL? Then this looks applicable. You can tell the AMD/nVidia card to automatically use the best card:

https://stackoverflow.com/questions/6036292/select-a-graphic-device-in-windows-opengl
```C
// enable optimus!
extern "C" {
    _declspec(dllexport) DWORD NvOptimusEnablement = 1;
    _declspec(dllexport) int AmdPowerXpressRequestHighPerformance = 1;
}
````

**Comment by shinyquagsire23** [comment]
oh interesting, I'll give that a stab then in the next release

**Comment by shinyquagsire23** [comment]
maybe fixed v0.7.0?
